crates/vm/src/system/memory/offline_checker/mod.rs

```diff
@@ -14,7 +14,9 @@ pub struct MemoryReadAuxRecord {
 
 #[repr(C)]
 #[derive(Debug, Clone)]
-pub struct MemoryWriteAuxRecord<const NUM_LIMBS: usize> {
+pub struct MemoryWriteAuxRecord<T, const NUM_LIMBS: usize> {
     pub prev_timestamp: u32,
-    pub prev_data: [u8; NUM_LIMBS],
+    pub prev_data: [T; NUM_LIMBS],
 }
+
+pub type MemoryWriteBytesAuxRecord<const NUM_LIMBS: usize> = MemoryWriteAuxRecord<u8, NUM_LIMBS>;
```

Comment:
renamed from MemoryWriteAuxRecord to MemoryWriteBytesAuxRecord in b04b4df0d3eab61013142fac517f370b621464ce